src/app/api/series/discover/route.ts

```
@@ -0,0 +1,41 @@
+
+        const response = await fetch(
+            `${BASE_URL}/discover/tv?api_key=${TMDB_API_KEY}&language=en-US&with_genres=${withGenres}&append_to_response=overview`,
+            { next: { revalidate: 60 * 60 * 12 } } // Cache for 12 hours
+        );
+
+        if (!response.ok) {
+            throw new Error('Failed to fetch data');
+        }
+
+        const data = await response.json();
+        const results = data.results.map((show: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)